taproot.html

```diff
@@ -408,40 +414,40 @@ <h2>Security notice</h2>
 <div style="clear: both;">&nbsp;</div>
 
 <span class="anchor" id="faq"></span>
-<div style="width: 80%; margin: auto; text-align: center; font-size: 150%;">
+<div style="width: 75%; margin: auto; text-align: center; font-size: 150%;">
 <h2 style="font-size: 200%;">FAQ</h2>
 
 <h3 style="font-size: 120%">Taproot Itself</h3>
 <dl>
 
-<dd><span class="anchor" id="faq_privacy"></span>How does Taproot improve privacy?</dd>
+<dd class="clickable"><span class="anchor" id="faq_privacy"></span>How does Taproot improve privacy?</dd>
```

Comment:
This should not be changed. The Javascript will add clickable if it is indeed clickable. Note that it *isn't* clickable in non-JS browsers.